includes/class-amp-theme-support.php

```diff
@@ -833,4 +834,20 @@ public static function prepare_response( $response, $args = array() ) {
 
 		return $response;
 	}
+
+	/**
+	 * Adds 'data-amp-layout' to the allowed <img> attributes for wp_kses().
+	 *
+	 * @since 0.7
+	 *
+	 * @param array $context Allowed tags and their allowed attributes.
+	 * @return array $context Filtered allowed tags and attributes.
+	 */
+	public static function add_layout( $context ) {
```

Comment:
I think this should be named a bit more explicitly, like: `whitelist_layout_in_wp_kses_allowed_html()`.

Reply:
Thanks, [this commit](https://github.com/Automattic/amp-wp/pull/937/commits/e344418132bc56fbf4d0c6083f399061f0ab4976) renames the function to `whitelist_layout_in_wp_kses_allowed_html()`.